modules/msk/gbcms/tests/main.nf.test

```diff
@@ -0,0 +1,45 @@
+// nf-core modules test gbcms
+nextflow_process {
+
+    name "Test Process GBCMS"
+    script "../main.nf"
+    process "GBCMS"
+    config "./nextflow.config"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "modules_msk"
+    tag "gbcms"
+
+    test("sarscov2 illumina  paired-end [fastq]") {
+
+        // (the module requires running more than one process to generate the required output)
+        // add the 'setup' method here.
+        // You can find more information about how to use a 'setup' method in the docs (https://nf-co.re/docs/contributing/modules#steps-for-creating-nf-test-for-chained-modules).
```

Comment:
seems like part of the TODO block, should we remove?

Reply:
I removed these